### ISSUE TITLE
Invalid syntax in generated typescript file

### ISSUE THREAD

**Issue by gabrielcolson** [ServerlessLife/serverless-spy#8]
I have a Lambda function in my stack declared this way: `new ln.NodeJsFunction(this, "custom-message", {...})`

it generates the following (invalid) typescript file:
```ts
/* eslint-disable */
export class ServerlessSpyEvents {
  Functioncustom-messageRequest: 'Function#custom-message#Request' = 'Function#custom-message#Request';
  Functioncustom-messageError: 'Function#custom-message#Error' = 'Function#custom-message#Error';
  Functioncustom-messageConsole: 'Function#custom-message#Console' = 'Function#custom-message#Console';
  Functioncustom-messageResponse: 'Function#custom-message#Response' = 'Function#custom-message#Response';
}
```

I think ServerlessSpy should convert the construct id to camel case. This will fix the typescript file and improve the readability of the `waitForXxx` functions

**Comment by ServerlessLife** [comment]
True. It is a bug. I will fix it over the weekend.

**Comment by ServerlessLife** [comment]
It is fixed (version 1.0.4)
